src/app/utils/device-screen-size.ts

```diff
@@ -0,0 +1,19 @@
+export function getDeviceScreenSize(deviceWidth: number){
+    if(deviceWidth >= 1920){
+      return 'xxxl';
+    }
+    else if(deviceWidth >= 1366){
+      return 'xxl';
+    }
+    else if(deviceWidth >= 1365){
```

Comment:
this conditions is incorrect
```suggestion
    else if(deviceWidth >= 1024){
```